### PR TITLE
Fix model

### PR DESCRIPTION
**Pull request by lukyanoffpashok**

### CODE REVIEW

**Comment by 0crat** [comment]
@lukyanoffpashok[/z](https://www.0crat.com/u/lukyanoffpashok) this pull request is too small, just 3 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @acheshkov[/z](https://www.0crat.com/u/acheshkov) please review this and merge or reject

File: aibolit/model/model.py [hunk 1]
```diff
@@ -47,12 +47,13 @@ def preprocess_file(
 
         df.drop('filename', axis=1, inplace=True)
         df.drop('index', axis=1, inplace=True)
-        self.target = df[['cyclo']].copy().values
+        self.target = df[['Cognitive Complexity']].values[:, 0]
```
Comment (lyriccoder) [inline aibolit/model/model.py:50]: we have M4 code for Cognitive Complexity, not 'Cognitive Complexity'

File: aibolit/model/model.py [hunk 2]
```diff
@@ -47,12 +47,13 @@ def preprocess_file(
...
         if scale_ncss:
             new = pd.DataFrame(
                 df[self.only_patterns].values / df['M2'].values.reshape((-1, 1)),
                 columns=self.only_patterns
             )
+            self.target /= df['M2'].values.reshape(-1)
```
Comment (lyriccoder) [inline aibolit/model/model.py:56]: why does we have reshape here?

File: aibolit/model/model.py (hunk 1 above)
Comment (lyriccoder) [inline aibolit/model/model.py:50]: why do we have .values[:, 0]

**Comment by acheshkov** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#314` is not assigned, can't get performer

**Comment by acheshkov** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/yegor256/aibolit/issues/314#issuecomment-629348828))

@acheshkov Job `gh:yegor256/aibolit#314` is not assigned, can't get performer

**Pull request merged**